python/setup.py.in

```diff
@@ -1,13 +1,12 @@
 #
-# Copyright (C) 2019-2021 VMware, Inc. All Rights Reserved.
+# Copyright (C) 2019-2022 VMware, Inc. All Rights Reserved.
 #
 # Licensed under the GNU General Public License v2 (the "License");
 # you may not use this file except in compliance with the License. The terms
 # of the License are located in the COPYING file of this distribution.
 #
 
-import os
-from distutils.core import setup
+from setuptools import setup
```

Comment:
This is a significant change, disutils is deprecated and will be removed from python-3.12